alpha_integration_dev/old_client.py

```diff
@@ -106,8 +106,11 @@ def log_series(self):
         neptune.log_image('g_img', self.img_path)
 
         # see what we've logged
+        from time import sleep
+        sleep(5)
```

Comment:
Is there any equivalent of `exp.sync()` in old client?

Reply:
Nope.